### ISSUE TITLE
Screen Reader Support Specification Adaptation - Slider/Range Slider components

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2664]
**Is your feature request related to a problem? Please describe.**
We should analyse the existing screen reader support implementation, along with the UI5 SR Specification itself and Aria Best Practices and adapt them in a way that fit the ui5 web components.

**Describe the solution you'd like**
Analyse and create a proposal specification for the slider and range slider components.The screen reader support implementation proposal, should be documented in the accessibility status wiki of the component.

**Comment by elenastoyanovaa** [comment]
The specification is created and the wiki is updated. The implementation will be done as part of https://github.com/SAP/ui5-webcomponents/issues/2513
